applications/nrf_desktop/src/modules/led_stream.c

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <misc/byteorder.h>
+
+#define MODULE led_stream
+#include "module_state_event.h"
+
+#include <logging/log.h>
+
+#include "config_event.h"
+#include "led_event.h"
+
+#define STREAM_LED_ID 0
+#define INCOMING_LED_COLOR_COUNT 3
```

Comment:
Maybe define this using Kconfig symbol: `CONFIG_DESKTOP_LED_COLOR_COUNT`. And add build assert that only 3 colors are supported.

Reply:
discussed offline

Reply:
actually there is no need for assert - you can simply make Kconfig option depend on this value